src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,18 @@
 
 public class MainApp {
 
+    public static void mainApp(String[] args) {
```

Comment:
The method name should be `main` instead of `mainApp`. The `main` method is the entry point of a Java application and should always be named as such.